exchange-common/src/main/scala/com/vesoft/exchange/common/Package.scala

```diff
@@ -97,4 +97,6 @@ final case class Argument(config: String = "application.conf",
                           hive: Boolean = false,
                           directly: Boolean = false,
                           dry: Boolean = false,
-                          reload: String = "")
+                          reload: String = "",
+                          file: Boolean = false,
+                          path: String = "")
```

Comment:
it's a great work, thanks!
maybe we should use more understandable variable name，how about change `file` to `variable`， `path` to `param`?